crates/rpc/rpc/src/trace.rs

```diff
@@ -46,7 +47,7 @@ impl<Eth> TraceApi<Eth> {
     /// Create a new instance of the [`TraceApi`]
     pub fn new(eth_api: Eth, blocking_task_guard: BlockingTaskGuard) -> Self {
         let inner = Arc::new(TraceApiInner { eth_api, blocking_task_guard });
-        Self { inner }
+        Self { inner, eth_config: EthConfig::default() }
```

Comment:
I don't think this works just yet, but nothing is being configured here and this will always use the default

Reply:
so what we need to do here is provide the ethconfig as arg, or just the max_trace_filter_blocks